lib/types.ts

```diff
@@ -705,6 +707,8 @@ export interface BaseExtensionSDK {
   access: AccessAPI
   /** Exposes relevant ids, keys may be ommited based on location */
   ids: IdsAPI
+  /** the adapter that can be used for creating the cma client */
```

Comment:
Not sure this comment is correct, the adapter is already created at the time we create the SDK, right?

Reply:
Yes, the adapter is created, but the comment says that you can use it to create the cma client, or did I word that wrong?🤔.

Reply:
The comment is correct but still should be reworded to be consistent with the other comments

Reply:
Updated the commment